README.md

```diff
@@ -129,3 +129,61 @@ Then you can connect to it programmatically using `mysql`:
 ```bash
 mysql -u oba_user -p -h localhost:3306
 ```
+
+### Using Google Maps
+
+#### Prerequisites
+
+- Have a valid Google Cloud Platform (GCP) account.
+- Create a project on GCP and enable the Maps JavaScript API for it.
+- Create an API key for your project. [See how to create an API key](https://cloud.google.com/docs/authentication/api-keys).
+- If there is a rate limit error while using the test API key, it means you have reached the maximum usage limit for that key, you need to generate a new API key.
+
+#### Configuring Google Maps
+
+You'll need to set the following environment variables:
+
+- `GOOGLE_MAPS_API_KEY`: Your Google Maps API key.
+- `GOOGLE_MAPS_CLIENT_ID`: (Optional) Required if you are using the Google Maps Premium Plan.
+- `GOOGLE_MAPS_CHANNEL_ID`: (Optional) A channel ID that helps identify the source of API requests for analytics and reporting purposes.
+
+#### Docker Configuration
+
+If using Docker Compose:
+
+1.Modify the 'docker-compose.yml' file like this:
+
+```yaml
+services:
+  oba-app:
+    build:
```

Comment:
these should be injected at runtime, not build time.

Reply:
Thank you, Aaron. I have already update to inject environment variables at runtime :D